### PR TITLE
[FRONT-1542] Streams service moved to Apollo client

### PR DESCRIPTION
**Pull request by pmichalowski**

### CODE REVIEW

File: src/pages/StreamListingPage.tsx [hunk 1]
```diff
@@ -59,37 +61,35 @@ const mapOrderByToIndexer = (orderBy: OrderBy): IndexerOrderBy => {
...
     switch (orderBy) {
-        case OrderBy.Id: {
-            return TheGraphOrderBy.Id
+        case ListOrderBy.Id: {
+            return Stream_OrderBy.Id
         }
         default:
-            return TheGraphOrderBy.Id
+            return Stream_OrderBy.Id
     }
```
Comment (mondoreale) [inline src/pages/StreamListingPage.tsx:80]: That's a fancy way of saying
```ts
return Stream_OrderBy.Id
```

File: src/services/streams.ts [hunk 2]
```diff
@@ -131,34 +131,17 @@ const prepareStreamResult = (
...
-                        }
-                    }
-                }
-            `,
+    const {
+        data: { streams },
+    } = await getGraphClient().query<GetStreamsQuery, GetStreamsQueryVariables>({
+        query: GetStreamsDocument,
+        variables: {
+            streamIds: streamIds,
```
Comment (mondoreale) [inline src/services/streams.ts:139]:
```ts
streamIds,
```

File: src/services/streams.ts [hunk 3]
```diff
@@ -169,60 +152,38 @@ export const getPagedStreams = async (
...
+    where = _.omitBy(where, _.isEmpty)
+
+    const {
+        data: { streams },
+    } = await getGraphClient().query<GetPagedStreamsQuery, GetPagedStreamsQueryVariables>(
+        {
+            query: GetPagedStreamsDocument,
+            variables: {
+                first: first + 1,
+                orderBy: orderBy ?? Stream_OrderBy.Id,
```
Comment (mondoreale) [inline src/services/streams.ts:178]: Any advantage of using `??` here?

File: src/services/streams.ts [hunk 4]
```diff
@@ -169,60 +152,38 @@ export const getPagedStreams = async (
...
+                orderDirection: orderDirection ?? OrderDirection.Asc,
```
Comment (mondoreale) [inline src/services/streams.ts:179]: And here?

File: src/services/streams.ts (hunk 3 above)
Comment (pmichalowski) [reply src/services/streams.ts:178]: yeah, those could be default param values

File: src/pages/StreamListingPage.tsx (hunk 1 above)
Comment (pmichalowski) [reply src/pages/StreamListingPage.tsx:80]: I'm guessing that whoever made this originally wanted to future-proof it

**Review by mondoreale — APPROVED** [review]
Yep, let's hit it.

File: src/pages/StreamListingPage.tsx (hunk 1 above)
Comment (mondoreale) [reply src/pages/StreamListingPage.tsx:80]: Looks like it only got worse. 😂

**Pull request merged**